### DISCUSSION TITLE
テストコードの中にある`cleanup()`は取り除くべきか？

### DISCUSSION THREAD

**Discussion by ShionTerunaga** [yamada-ui/yamada-ui#1647] [Ideas]
# 概要

先日記念すべき第一回`Yamada UI`の勉強会に参加をしました。
内容は[こちら](https://qiita.com/kobanyan/items/126512ec3e8d76c538b3)の輪読会でしたが、勉強会後に`Yamada UI`のテストコードを参照したら`cleanup()`を記載されたテストコードを発見しました。

取り除いてもテストは通るので上記のルール的に取り除いた方が良いのかなと思ったのでこちらに記載しました。

# 対象のテストコード

- `packages/components/slider/tests/range-slider.test.tsx`
- `packages/components/toggle/tests/toggle-group.test.tsx`
- `packages/components/toggle/tests/toggle.test.tsx`

**Comment by 108yen** [comment]
消していいと思います。
もう一度レンダリングしたいなら`rerender`使うべきです。

**Reply by ShionTerunaga** [reply]
@108yen 
  
回答ありがとうございます。

@hirotomoyamada

#1648 
#1649 
#1650 

にてissueを立てさせていただきました。
